frmts/pdf/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ target_include_directories(gdal_PDF PRIVATE ${GDAL_RASTER_FORMAT_SOURCE_DIR}/vrt
                                             ${GDAL_VECTOR_FORMAT_SOURCE_DIR}/mem)
 
 if (GDAL_USE_POPPLER)
-  target_include_directories(gdal_PDF PRIVATE ${POPPLER_INCLUDE_DIRS} ${POPPLER_INCLUDE_DIRS}/../)
+  target_include_directories(gdal_PDF PRIVATE ${POPPLER_INCLUDE_DIR} ${POPPLER_INCLUDE_DIR}/../)
```

Comment:
I think this should be fixed in the Find module. Recommended pattern:
`<Pkg>_INCLUDE_DIR` is an input variable (for the user).
`<Pkg>_INCLUDE_DIRS` is the output variable (for the consuming project).
(And `<Pkg>` spelled as in `FindPkg.cmake`.)

Reference:
https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#standard-variable-names

Reply:
> I think this should be fixed in the Find module. 

renaming FindPoppler.cmake to FindPOPPLER.cmake and keeping the use of POPPLER_INCLUDE_DIRS ?

Reply:
I see `_INCLUDE_DIRS` as a strict requirement.

I am more relaxed about accepting upper case also for FindPoppler.cmake. I don't think we will see a `FindPoppler.cmake` from CMake. (But an exported config from Poppler would be nice.) 
The coincidence is  that I just stumbled over Poppler using an early version of `FindFontconfig.cmake`, before https://github.com/Kitware/CMake/commit/a3a1e69f8a8e6571bbb8c9257b401f04c22bd1e2. Now I need [extra polyfill](https://github.com/microsoft/vcpkg/blob/a8af4546305610346facda637b77f08e38166570/ports/poppler/cmake-project-include.cmake#L3-L6) to make Poppler use the static usage requirements vcpkg would offer through proper `Fontconfig_...` variables.